src/components/table-of-contents-controller/table-of-contents-controller.tsx

```diff
@@ -54,34 +66,42 @@ export class TableOfContentsController {
     return (
       <Host>
         <div class="search">
-          <input class={`search__input ${this.searchFilterInputClasses}`}
-                 value={this.titleFilter}
-                 placeholder={this.titleFilterPlaceholder}
-                 onInput={e => this.handleTitleChange(e)}/>
+          <input
+            class={`search__input ${this.searchFilterInputClasses}`}
+            value={this.titleFilter}
+            placeholder={this.titleFilterPlaceholder}
+            onInput={e => this.handleTitleChange(e)}
```

Comment:
As discussed, let's standardise on this calling format so that handleTitleChange doesn't have to be declared differently to everything else.

Reply:
done